apps/client/src/components/Modals/ManageReposTopicsModal.svelte

```diff
@@ -1,11 +1,10 @@
 <script lang="ts">
-  import { UPDATE_USER_TOPICS } from '$lib/graphql/mutations/users';
-  import { onDestroy } from 'svelte';
-  import { mutation } from 'svelte-apollo';
+  import { onDestroy, createEventDispatcher } from 'svelte';
   import { Button, Modal } from 'svelte-ui';
   import Card from 'svelte-ui/components/Card.svelte';
   export let user: any;
   export let topics: [];
+  export let isProcessing: boolean = false;
```

Comment:
A boolean is fine but I'd recommend something like an enum or a map as it'll help handle different states. What happens if a request fails?
For instance, say we have an enum `enum RequestState  { Processing, Completed, Failed }`

You could then use different calls to action i.e Failed = "try again" with a different handler.